arex-schedule-web-api/src/main/java/com/arextest/schedule/client/ZstdJacksonMessageConverter.java

```diff
@@ -41,6 +41,6 @@ protected Object readInternal(Class<?> clazz, HttpInputMessage inputMessage) thr
   @Override
   protected void writeInternal(Object o, HttpOutputMessage outputMessage) throws IOException,
       HttpMessageNotWritableException {
-    zstdJacksonSerializer.serializeTo(o, outputMessage.getBody());
+    outputMessage.getBody().write(zstdJacksonSerializer.serialize(o));
```

Comment:
What is the purpose of the change codes here？

Reply:
outputMessage.body is OutputStream，i need ByteArrayOutputStream to echo the arex-storage project